skimage/color/colorconv.py

```diff
@@ -824,8 +824,12 @@ def gray2rgb(image, alpha=None):
     Raises
     ------
     ValueError
-        If the input is not a 2- or 3-dimensional image.
+        If the input is not a 1-, 2- or 3-dimensional image.
 
+    Notes
+    -----
+    If the input is a 1-dimensional image of shape ``(M, )``, the output
+    will be shape ``(M, 1, 3)``.
```

Comment:
Did I miss a discussion as to why this is the case? I would certainly find it much more intuitive if the return value was of shape `input.shape + (3,)` in all cases — even higher-dimensional images.